### PR TITLE
Include parent entity name in payloads

### PR DESCRIPTION
**Pull request by matheushent**
#### What
* GET /job-scripts returns the application name for each entity returned
* GET /job-scripts/<id> returns the application name of the job script whose id is <id>
* GET/job-submissions returns the job script id for each entity returned
* GET/job-submissions/<id> returns the job script id of the job submission whose id is <id>

#### Why
Enable faster querying against the Jobbergate API

`Task`: https://app.clickup.com/t/18022949/PFT-1086 (private task)

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: jobbergate-api/jobbergate_api/apps/job_scripts/routers.py [hunk 1]
```diff
@@ -115,7 +116,7 @@ async def job_script_create(
     logger.debug(f"Job-script created: {dict(job_script_data)}")
 
     response = JobScriptResponse(
-        **job_script_data,
+        **{**job_script_data, "application_name": application.application_name},
```
Comment (fschuch) [inline jobbergate-api/jobbergate_api/apps/job_scripts/routers.py:119]: If you agree, we can achieve the same effect with a better readability:

```suggestion
        **job_script_data,
        application_name=application.application_name,
```

File: jobbergate-api/jobbergate_api/apps/job_submissions/routers.py [hunk 2]
```diff
@@ -115,7 +115,9 @@ async def job_submission_create(
 
     logger.debug(f"Job-submission created: {job_submission_data=}")
 
-    return JobSubmissionResponse(**job_submission_data)  # type: ignore
+    return JobSubmissionResponse(
+        **{**job_submission_data, "job_script_name": raw_job_script.get("job_script_name")}  # type: ignore
```
Comment (fschuch) [inline jobbergate-api/jobbergate_api/apps/job_submissions/routers.py:119]: If you agree:

```suggestion
        **job_submission_data, job_script_name=raw_job_script.get("job_script_name")  # type: ignore
```

**Review by fschuch — CHANGES_REQUESTED** [review]
Good work, I left a few comments for you.

Also, there are no changes on the tests for job-scripts, is that correct?

File: jobbergate-api/jobbergate_api/apps/job_scripts/routers.py (hunk 1 above)
Comment (dusktreader) [reply jobbergate-api/jobbergate_api/apps/job_scripts/routers.py:119]: Actually, this won't work if `job_script_data` already includes `application_name`. Python will complain about duplicate keys.

**Comment by matheushent** [comment]
> Good work, I left a few comments for you.
> 
> Also, there are no changes on the tests for job-scripts, is that correct?

Correct, no changes on the tests for job scripts

File: jobbergate-api/jobbergate_api/apps/job_scripts/routers.py (hunk 1 above)
Comment (fschuch) [reply jobbergate-api/jobbergate_api/apps/job_scripts/routers.py]: I see your point. @matheushent, feel free to ignore both of my code suggestions in this PR.

**Review by fschuch — APPROVED** [review]

File: jobbergate-api/jobbergate_api/apps/job_scripts/routers.py (hunk 1 above)
Comment (matheushent) [reply jobbergate-api/jobbergate_api/apps/job_scripts/routers.py]: > I see your point. @matheushent, feel free to ignore both of my code suggestions in this PR.

well, I implemented your suggestions and the tests passed.

Comment (fschuch) [reply jobbergate-api/jobbergate_api/apps/job_scripts/routers.py]: Yes, I've checked there, `job_script_data` is not expected to include `application_name`.

**Comment by dusktreader** [comment]
Task linked: [PFT-1086 Backend: Include parent entity name in payloads](https://app.clickup.com/t/18022949/PFT-1086)

**Comment by dusktreader** [comment]
Task linked: [PFT-1086 Backend: Include parent entity name in payloads](https://app.clickup.com/t/18022949/PFT-1086)

**Pull request merged**